src/OrchardCore/OrchardCore.DisplayManagement/OrchardCore.DisplayManagement.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.2</TargetFrameworks>
```

Comment:
Because there's an obscure in the .NET SDK that causes a super weird error I spent 3 hours to debug (unsuccessfully). It doesn't hurt so we can still take a look later.

Reply:
https://ci.appveyor.com/project/alexbocharov/orchard2/builds/21559582#L1014

It really makes no sense because this project - like all the other ones - targets `.NET Standard 2.0`, so it should be usable by any project that has both `netstandard2.0` and `netcoreapp2.2`.

Do you know someone from the SDK/NuGet teams we could ping to shed some light on that?